README.md

```diff
@@ -1089,6 +1089,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [Bitwarden](https://bitwarden.com/) `⚠` - Password manager with webapp, browser extension, and mobile app. ([Source Code](https://github.com/bitwarden/core)) `AGPL-3.0` `C#`
 - [bitwarden_rs](https://github.com/dani-garcia/bitwarden_rs) - Lightweight Bitwarden server API implementation written in Rust. `GPL-3.0` `Rust`
 - [keeweb](https://keeweb.info/) - This webapp is a browser and desktop password manager compatible with KeePass databases. ([Source Code](https://github.com/keeweb/keeweb)) `MIT` `HTML5`
+- [Laravel Shaarli](https://github.com/MarceauKa/laravel-shaarli) - All in one plateform for your links, stories and passwords. Built with Laravel and Vue.js. ([Demo](http://shaarli.mka.ovh/), [Source Code](https://github.com/MarceauKa/laravel-shaarli)) `MIT` `PHP`
```

Comment:
Thank you for your contribution.
Please remove the source code link from the end and add the demo https://shaarli.mka.ovh